### PR TITLE
citra_qt/main.ui: remove unused actions "Load Symbol Map..." and "Select Game Directory..."

### PR DESCRIPTION
**Pull request by vvanelslande**
Self-explanatory. No UI changes.

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/4953)

### CODE REVIEW

File: src/citra_qt/main.ui [hunk 1]
```diff
@@ -45,7 +45,7 @@
      <x>0</x>
      <y>0</y>
      <width>1081</width>
-     <height>21</height>
+     <height>22</height>
```
Comment (zhaowenlan1779) [inline src/citra_qt/main.ui]: Is this intentional?

Comment (vvanelslande) [reply src/citra_qt/main.ui]: Qt Designer changed this

Comment (zhaowenlan1779) [reply src/citra_qt/main.ui]: That doesn't mean it *should* be changed though

**Comment by vvanelslande** [comment]
Reverted the `height` change.

**Pull request merged**